fix/mab2De-Sol1Holdings_seq2strapi.fix

```diff
@@ -0,0 +1,124 @@
+	put_filemap("../prod/map/identifier.tsv","hbzId2AlmaMmsId", sep_char:"\t",key_column:"1",value_column:"0",expected_columns:"-1")
+	put_filemap("../prod/map/identifier.tsv","zdbId2AlmaMmsId", sep_char:"\t",key_column:"2",value_column:"0",expected_columns:"-1")
+	put_filemap("../prod/map/identifier.tsv","hbzId2zdbId", sep_char:"\t",key_column:"1",value_column:"2",expected_columns:"-1")
+
+	do put_macro("dateNormalizer")
+		replace_all("$[field]", "(\\d{4})(\\d{2})(\\d{2})","$1-$2-$3")
+	end
+
+	do put_macro("moveToNote")
+		if exists("$[field]")
+			paste("$[targedField]","~$[field]:", "$[field].a")
+		end
+	end
+end
+
+if any_equal("FMT","ML")
+
+	copy_field("001.a","holdingId")
+	copy_field("002a.a","dateCreated")
+	copy_field("003.a","dateModified")
+	copy_field("004.a","dateAustausch")
+	# 012.a
+
+	copy_field("012.a","hbzId")
+	copy_field("012.a","almaMmsId")
+	copy_field("012.a","zdbId")
+
+	# 210a.*.d
+	# 210a.*.j
+	# 210a.*.k
+	# 210a.*.n
+	# 210a.d
+	# 210a.j
+	# 210a.k
+	# 210a.k.*
+	# 210a.n
+	# 210b.d
+	# 210b.j
+	# 210b.j.*
+	
+	do list(path:"210a","var":"$i")
+		copy_field("$i.d","@210a.$append")
+		copy_field("$i.n","@210a.$append")
+		if is_array("$i.k")
+			join_field("$i.k",", ")
+		end
+		copy_field("$i.k","@210a.$append")
+		if is_array("$i.j")
+			join_field("$i.j",", ")
+		end
+		copy_field("$i.j","@210a.$append")
+		join_field("@210a",", ")
+		paste("holdingInfo.$append","~210a: ", "@210a")
+	end
+	do list(path:"210b","var":"$i")
+		copy_field("$i.d","@210b.$append")
+		copy_field("$i.n","@210b.$append")
+		if is_array("$i.k")
+			join_field("$i.k",", ")
+		end
+		copy_field("$i.k","@210b.$append")
+		if is_array("$i.j")
+			join_field("$i.j",", ")
+		end
+		copy_field("$i.j","@210b.$append")
+		join_field("@210b",", ")
+		paste("holdingInfo.$append","~210b: ", "@210b")
+	end
+	call_macro("moveToNote",field:"220",targedField:"holdingInfo.$append")
+	join_field("holdingInfo","\n")
+
+else
+	copy_field("001.a","item.itemId")
+	copy_field("002a.a","item.dateCreated")
+	copy_field("003.a","item.dateModified")
+	copy_field("004.a","item.dateAustausch")
+# 010.a
+# 012.a
+	# copy_field("012.a","hbzId")
+	copy_field("012.a","almaMmsId")
+	copy_field("012.a","zdbId")
+	lookup("zdbId", "hbzId2zdbId",delete:"true")
+
+	copy_field("100.a","item.callNumber")
+
+	copy_field("115.a","item.inventoryNumber")
+
+	call_macro("moveToNote",field:"125a",targedField:"item.note.$append")
+	call_macro("moveToNote",field:"125b",targedField:"item.note.$append")
+	call_macro("moveToNote",field:"107",targedField:"item.note.$append")
+	call_macro("moveToNote",field:"120",targedField:"item.note.$append")
+	call_macro("moveToNote",field:"105",targedField:"item.note.$append")
+	join_field("item.note","\n")
+
+end
+
+lookup("zdbId", "hbzId2zdbId",delete:"true")
+
+if any_match("almaMmsId","^[A-Z]{2}.*")
+	lookup("almaMmsId","hbzId2AlmaMmsId")
+else
+	lookup("almaMmsId","zdbId2AlmaMmsId")
+end
+
+call_macro("dateNormalizer",field:"dateCreated")
```

Comment:
For consistency with the verb style of Fix functions, I think this macro should be called `normalizeDate`.

(And for consistent naming, I think the macros should use underscores instead of camel case.)

Reply:
It's my suggestion, to use e.g. `normalize_date`, to have it look like functions e.g. `add_field`, etc.

Reply:
ah, okay, interesting view, if often use these for repeated fix processes that are not so easy to pin down. I will try.